src/ert/simulator/simulation_context.py

```diff
@@ -152,26 +152,11 @@ def isRunning(self) -> bool:
         # TODO: Should separate between running jobs and having loaded all data
         return self._sim_thread.is_alive() or self._job_queue.is_active()
 
-    def getNumPending(self) -> int:
-        return self._job_queue.count_status(JobStatus.PENDING)  # type: ignore
-
-    def getNumRunning(self) -> int:
-        return self._job_queue.count_status(JobStatus.RUNNING)  # type: ignore
-
-    def getNumSuccess(self) -> int:
-        return self._job_queue.count_status(JobStatus.SUCCESS)  # type: ignore
-
-    def getNumFailed(self) -> int:
-        return self._job_queue.count_status(JobStatus.FAILED)  # type: ignore
-
-    def getNumWaiting(self) -> int:
-        return self._job_queue.count_status(JobStatus.WAITING)  # type: ignore
-
     def didRealizationSucceed(self, iens: int) -> bool:
```

Comment:
Do we really want Camelcase function names?

Reply:
Yes, there are some other functions in `simulation_context` using Camelcase, but I would keep the current API names unchanged.